docs/expo.md

```diff
@@ -0,0 +1,56 @@
+# Integrating the SDK with Expo projects
+
+- [Guide for Expo apps](#guide-for-expo-apps)
+- [Guide for Bare React Native apps using Expo modules](#guide-for-bare-react-native-apps-using-expo-modules)
+
+## Guide for Expo apps
+
+### Overview
+Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a development build with Expo CLI allows you to include your specific native dependencies and customizations.
+
+When initializing the Adobe Mobile SDKs, it is required to initialize the SDKs in the native code inside your AppDelegate file for iOS and the MainApplication file for Android.
+
+By default, Expo projects use Continuous Native Generation (CNG). This means that projects do not have android and ios directories containing the native code and configuration. You can opt out of CNG and directly manage the code and configuration inside your android and ios directories.
+
+### Installation
+
+- To generate these directories, run `npx expo prebuild` or compile your app locally (`npx expo run:android` or `npx expo run:ios`).
+
```

Comment:
How about change to this format?

### Installation

To generate these directories, run

```bash
// create android and iOS directories
npx expo prebuild
 ```
 
 Or compile your app locally
 
 ```bash
 # Build your native Android project
 npx expo run:android
 # Build your native iOS project
 npx expo run:ios
 ```